cps/CP21.md

```diff
@@ -0,0 +1,50 @@
+---
+cp: 16
+title: 2022-12-16 Updating Tinlake Rewards Allocation
+authors: Governance and Coordination Group (ImdioR, Orhan)
+contributors: Governance and Coordination Group (ImdioR, Orhan)
+proposal-type: CP-4
+date-proposed: 2022-12-16
+status: discussion

```

Comment:
status will be "polling" by the time this PR is merged

Reply:
++